boto3/docs/action.py

```diff
@@ -29,6 +29,32 @@
     get_resource_public_actions,
 )
 
+WARNING_MESSAGE_TEMPLATE = """
+.. warning::
+    Please note that this method does not function as expected. It is
+    recommended to use the :py:meth:`{suggested_py_method_name}`
+    :doc:`client method <{suggested_py_method_rel_path}>` instead.
+    {extra}
+"""
+DOCUMENT_ACTION_OVERRIDES = {
+    "Metric": {
+        "put_data": {
+            "ignore_params": ["Namespace"],
+            "warning": WARNING_MESSAGE_TEMPLATE.format(
+                suggested_py_method_name="put_metric_data",
+                suggested_py_method_rel_path=(
+                    "../../cloudwatch/client/put_metric_data"
+                ),
+                extra=(
+                    "If you would still like to use this method, please make sure "
+                    "that `MetricData[].MetricName` is equal to the metric resource's "
+                    "`name` attribute."
```

Comment:
nit: regarding the line with ``` "`name` attribute."```, the screenshot shows it without the `.` at the end. Was this added after the screenshot was taken? Or is there an issue with the period?